From 8a2e280fb675636124fa9e217313b26c1ad796a0 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 23 Jul 2013 13:05:26 +0800
Subject: [PATCH 458/635] ENGR00271827 wifi:make atheros and realtek work in
 one image

For atheros wifi driver, change rules to make it and realtek can
work in one image.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 wlan/Android.mk                         | 2 +-
 wlan/atheros/compat-wireless/Android.mk | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/wlan/Android.mk b/wlan/Android.mk
index 1cb4fac..5fb1071 100644
--- a/wlan/Android.mk
+++ b/wlan/Android.mk
@@ -1,4 +1,4 @@
 atheros_dirs := compat-wireless
-ifeq ($(BOARD_HAS_ATH_WLAN),true)
+ifeq ($(BOARD_WLAN_DEVICE),$(filter $(BOARD_WLAN_DEVICE),ar6003 UNITE))
     include $(call all-subdir-makefiles,$(atheros_dirs))
 endif
diff --git a/wlan/atheros/compat-wireless/Android.mk b/wlan/atheros/compat-wireless/Android.mk
index ed2497c..370e305 100644
--- a/wlan/atheros/compat-wireless/Android.mk
+++ b/wlan/atheros/compat-wireless/Android.mk
@@ -22,7 +22,6 @@
 #==============================================================================
 
 ifneq ($(TARGET_SIMULATOR),true)
-ifeq ($(BOARD_WLAN_DEVICE),ar6003)
 
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
@@ -64,4 +63,3 @@ LOCAL_SRC_FILES := check_wifi_mac.sh
 include $(BUILD_PREBUILT)
 
 endif
-endif
-- 
1.8.0

